### ISSUE TITLE
Pimatic won't start after homeduino update and init-d config

### ISSUE THREAD

**Issue by DavidZH** [pimatic/pimatic#641]
I made a grave error: I changed 2 things in pimatic in one go, and now it won't start anymore.
First I updated homeduino through the GUI, but I didn't restart. In stead I stopped and added the init-d script. (I haven't had need for that till now. I'm going away for a month, and my girlfriend and Linux.....) 

this is the error report:
```
14:45:56.777 [pimatic] error:>    at Framework._loadConfig (/pimatic-app/node_modules/pimatic/lib/framework.coffee:97:36)
14:45:56.777 [pimatic] error:>    at new Framework (/pimatic-app/node_modules/pimatic/lib/framework.coffee:55:8)
14:45:56.777 [pimatic] error:>    at /pimatic-app/node_modules/pimatic/startup.coffee:48:21
14:45:56.777 [pimatic] error:>    at tryCatcher (/pimatic-app/node_modules/pimatic/node_modules/bluebird/js/main/util.js:24:31)
14:45:56.777 [pimatic] error:>    at Function.Promise.attempt.Promise.try (/pimatic-app/node_modules/pimatic/node_modules/bluebird/js/main/method.js:31:24)
14:45:56.777 [pimatic] error:>    at Object.module.exports.startup (/pimatic-app/node_modules/pimatic/startup.coffee:47:10)
14:45:56.777 [pimatic] error:>    at run (/pimatic-app/node_modules/pimatic/pimatic.js:15:24)
14:45:56.777 [pimatic] error:>    at /pimatic-app/node_modules/pimatic/lib/daemon.coffee:87:9
14:45:56.777 [pimatic] error:>    at Object.oncomplete (fs.js:107:15)
14:45:56.792 [pimatic] info: exiting...

```
I tried reinstalling pimatic. No avail.
Error after reinstalling:
```
20:55:55.386 [pimatic] Startup error: SyntaxError: Unexpected token 
20:55:55.386 [pimatic]>
20:55:55.386 [pimatic]>    at Object.parse (native)
20:55:55.386 [pimatic]>    at Object.fStringDouble [as f] (/pimatic-app/node_modules/pimatic/node_modules/relaxed-json/relaxed-json.js:109:19)
20:55:55.386 [pimatic]>    at /pimatic-app/node_modules/pimatic/node_modules/relaxed-json/relaxed-json.js:58:34
20:55:55.386 [pimatic]>    at some (/pimatic-app/node_modules/pimatic/node_modules/relaxed-json/relaxed-json.js:37:13)
20:55:55.386 [pimatic]>    at findToken (/pimatic-app/node_modules/pimatic/node_modules/relaxed-json/relaxed-json.js:51:16)
20:55:55.386 [pimatic]>    at /pimatic-app/node_modules/pimatic/node_modules/relaxed-json/relaxed-json.js:65:23
20:55:55.386 [pimatic]>    at Object.transform (/pimatic-app/node_modules/pimatic/node_modules/relaxed-json/relaxed-json.js:224:18)
20:55:55.386 [pimatic]>    at Framework._loadConfig (/pimatic-app/node_modules/pimatic/lib/framework.coffee:97:36)
20:55:55.386 [pimatic]>    at new Framework (/pimatic-app/node_modules/pimatic/lib/framework.coffee:55:8)
20:55:55.386 [pimatic]>    at /pimatic-app/node_modules/pimatic/startup.coffee:48:21
20:55:55.386 [pimatic]>    at tryCatcher (/pimatic-app/node_modules/pimatic/node_modules/bluebird/js/main/util.js:24:31)
20:55:55.386 [pimatic]>    at Function.Promise.attempt.Promise.try (/pimatic-app/node_modules/pimatic/node_modules/bluebird/js/main/method.js:31:24)
20:55:55.386 [pimatic]>    at Object.module.exports.startup (/pimatic-app/node_modules/pimatic/startup.coffee:47:10)
20:55:55.386 [pimatic]>    at run (/pimatic-app/node_modules/pimatic/pimatic.js:15:24)
20:55:55.386 [pimatic]>    at Object.<anonymous> (/pimatic-app/node_modules/pimatic/pimatic.js:20:3)
20:55:55.386 [pimatic]>    at Module._compile (module.js:456:26)
20:55:55.386 [pimatic]>    at Object.Module._extensions..js (module.js:474:10)
20:55:55.386 [pimatic]>    at Module.load (module.js:356:32)
20:55:55.386 [pimatic]>    at Function.Module._load (module.js:312:12)
20:55:55.386 [pimatic]>    at Function.Module.runMain (module.js:497:10)
20:55:55.386 [pimatic]>    at startup (node.js:119:16)
20:55:55.386 [pimatic]>    at node.js:902:3
20:55:55.422 [pimatic] exiting...
```
I'm running Raspbian 3.12.26

**Comment by sweetpi** [comment]
That mean, that there's something wrong with your config file. Did you check the content and validate that it is valid json?

**Comment by DavidZH** [comment]
Aw hell! You're absolutely right.I deleted a device, and to avoid error messages I edited three rules where that device was liked to. Only I removed too much stuff from the end so the json was not valid anymore.

I promised to try to add value to this project, but me being busy has thwarted that completely for the last two months. Tat also means I didn't do any json typing so that got rusty. Mea culpa.
